aurora/main.tf

```diff
@@ -0,0 +1,75 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_db_subnet_group" "aurora" {
+  name        = "${var.project}-${var.environment}${var.tag}-aurora"
+  description = "Our main group of subnets"
+  subnet_ids  = ["${var.subnets}"]
+}
+
+resource "aws_rds_cluster_parameter_group" "aurora_mysql" {
+  name        = "mysql-rds-${var.project}-${var.environment}${var.tag}"
+  family      = "aurora5.6"
+  description = "rds ${var.project} ${var.environment} parameter group for mysql"
+
+  parameter = {
+    name = "slow_query_log"
+
+    value = "1"
+  }
+
+  parameter = {
+    name = "long_query_time"
+
+    value = "1"
+  }
+
+  parameter = {
+    name = "general_log"
+
+    value = "0"
+  }
+
+  parameter = {
+    name = "log_output"
+
+    value = "FILE"
+  }
+}
+
+resource "aws_rds_cluster" "aurora" {
```

Comment:
With the upcoming PostgreSQL support for Aurora, can you allow the `engine` and `engine_version` to be specified too?

https://aws.amazon.com/blogs/aws/amazon-aurora-update-postgresql-compatibility/